### PR TITLE
Support single showroom item.

### PR DESCRIPTION
**Pull request by bierik**
Fixes https://github.com/4teamwork/opengever.core/issues/1836

Add support for single showroom item.

### CODE REVIEW

**Comment by deiferni** [comment]
@bierik until the release is made, could you add `ftw.showroom` to the `development-packages` list in `sources.cfg`.

then i can merge 😉

File: opengever/bumblebee/browser/resources/bumblebee.js [hunk 1]
```diff
@@ -46,7 +46,11 @@
 
   function init() {
     showroom = Showroom([], { 'tail': tail });
-    updateShowroom();
+    if($(".template-search").length) {
```
Comment (deiferni) [inline opengever/bumblebee/browser/resources/bumblebee.js]:
```js
if($(".template-search").length || $(".portaltype-opengever-document-document").length)
 ...
```

File: opengever/bumblebee/browser/resources/bumblebee.js [hunk 2]
```diff
@@ -89,6 +93,10 @@
   }
 
   function updateShowroom() {
+    if($(".portaltype-opengever-document-document").length) {
```
Comment (deiferni) [inline opengever/bumblebee/browser/resources/bumblebee.js]: move this to `init`, see other comment

**Comment by bierik** [comment]
@deiferni updated

**Comment by deiferni** [comment]
👍

**Pull request merged**